flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/PartitionCommitPolicyFactory.java

```diff
@@ -63,11 +70,26 @@ public List<PartitionCommitPolicy> createPolicyChain(
                                             successFileName, fsSupplier.get());
                                 case PartitionCommitPolicy.CUSTOM:
                                     try {
-                                        return (PartitionCommitPolicy)
-                                                cl.loadClass(customClass).newInstance();
+                                        if (!CollectionUtil.isNullOrEmpty(parameters)) {
```

Comment:
The ci fail in here. 
```
Jun 29 04:30:25 04:30:25.403 [ERROR]   Architecture Violation [Priority: MEDIUM] - Rule 'Connector production code must depend only on public API when outside of connector packages' was violated (1 times):
Jun 29 04:30:25 Method <org.apache.flink.connector.file.table.PartitionCommitPolicyFactory.createPolicyChain(java.lang.ClassLoader, java.util.function.Supplier)> calls method <org.apache.flink.util.CollectionUtil.isNullOrEmpty(java.util.Collection)> in (PartitionCommitPolicyFactory.java:73)
```
Seems we can't call `CollectionUtil.isNullOrEmpty` in here.

Reply:
```
parameters != null && !parameters.isEmpty()
```

Reply:
Ok, and where can I see this ci fail? I hope to learn further : )

Reply:
https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=50618&view=results

Reply:
You can see the it in `CI report:`